File: VB/DXWebApplication1/DXWebApplication1/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace DXWebApplication1.Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!"

            Return View()
        End Function

        Private db As New DXWebApplication1.Models.Nwind_SalesDataEntities()

        <ValidateInput(False)> _
        Public Function PivotGridPartial() As ActionResult
            Dim model = db.SalesPersons
            Return PartialView("_PivotGridPartial", model.ToList())
        End Function
    End Class
End Namespace

File: CS/DXWebApplication1/Controllers/HomeController.cs
using DevExpress.Web.Mvc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace DXWebApplication1.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!";

            return View();
        }

        DXWebApplication1.Models.Nwind_SalesDataEntities db = new DXWebApplication1.Models.Nwind_SalesDataEntities();

        [ValidateInput(false)]
        public ActionResult PivotGridPartial()
        {
            var model = db.SalesPersons;
            return PartialView("_PivotGridPartial", model.ToList());
        }
    }
}